app/src/main/kotlin/io/imhungry/model/TagEntity.kt

```diff
@@ -0,0 +1,10 @@
+package io.imhungry.model
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "tag")
```

Comment:
The table name should be more descriptive. Perhaps something like restaurantTags